### PR TITLE
Br/add difftool formatter documentation.w 7836753

### PR DESCRIPTION
**Pull request by bredmond-sf**

### CODE REVIEW

**Review by unandyala — APPROVED** [review]

File: docs/diff/formatters.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Formatting the Diff Output
+
+The default output format for the diff tool is JSON. For a human-readable format, you can use `toFormattedString`.
```
Comment (bethlew) [inline docs/diff/formatters.md:3]: Suggest to "JSON is the default output format. Use `toFormattedStrong`' to view on the user console.

File: docs/diff/formatters.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+```typescript
+import { ApiChanges } from "@commerce-apps/raml-toolkit/lib/diff/changes/apiChanges";
+import { ApiDifferencer } from "@commerce-apps/raml-toolkit/lib/diff/apiDifferencer";
+
+const apiDifferencer = new ApiDifferencer("diff/left.raml", "diff/right.raml");
+
+apiDifferencer.findAndCategorizeChanges().then((changes: ApiChanges) => {
+    // Default output format is JSON
```
Comment (bethlew) [inline docs/diff/formatters.md:12]: JSON is default output format

File: docs/diff/formatters.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    console.log(changes);
+    // Use human-readable console format
```
Comment (bethlew) [inline docs/diff/formatters.md:14]: Use to view on the user console.

File: docs/diff/formatters.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    console.log(changes.toFormattedString("console"));
+});
+```
+
+The output is formatting using Handlebars templates. To customize the output, copy the template you'd like to customize from `resources/diff/templates`, modify it and apply it as shown here.
```
Comment (bethlew) [inline docs/diff/formatters.md:19]: Handlebars templates format the output. To customize the output, copy the template in `resources/diff/templates`, modify it, and apply it as shown here.

**Review by bethlew — DISMISSED** [review]

**Review by bethlew — APPROVED** [review]

**Review by gkhunger-sf — APPROVED** [review]

**Pull request merged**